x-pack/plugins/cases/server/client/configure/client.ts

```diff
@@ -288,15 +284,13 @@ async function update(
     let error = null;
     const updateDate = new Date().toISOString();
     let mappings: ConnectorMappingsAttributes[] = [];
-    const { connector, ...queryWithoutVersionAndConnector } = queryWithoutVersion;
+    const { connector } = queryWithoutVersion;
 
     try {
       const resMappings = await casesClientInternal.configuration.getMappings({
-        connector:
-          connector != null
-            ? connector
-            : transformESConnectorToCaseConnector(configuration.attributes.connector),
```

Comment:
We no longer need to do the transformation here because that functionality/responsibility has been moved to the service layer.

The transformations happen a couple times in this file, so I won't repeat this message here haha.